<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsPage</class>
 <widget class="QWidget" name="settingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>430</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Step 3: Select application settings</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Please select the settings for your requested application:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="settingsStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Database</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Select the database you want to import from:</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="databaseBox"/>
          </item>
          <item>
           <widget class="QStackedWidget" name="databaseStack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="sqlitePage">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Configure SQLite</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="margin">
                  <number>6</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Filename:</string>
                     </property>
                     <property name="buddy">
                      <cstring>sqlite_fileLine</cstring>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="sqlite_fileLine"/>
                   </item>
                   <item>
                    <widget class="QToolButton" name="openFileButton">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="resource.qrc">
                       <normaloff>:/document-open</normaloff>:/document-open</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>148</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="mysqlPage">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Configure MySQL</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="margin">
                  <number>6</number>
                 </property>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="mysql_userLine"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>User:</string>
                   </property>
                   <property name="buddy">
                    <cstring>mysql_userLine</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2" rowspan="3">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Preferred</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>10</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="3">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Port:</string>
                   </property>
                   <property name="buddy">
                    <cstring>mysql_portBox</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Database:</string>
                   </property>
                   <property name="buddy">
                    <cstring>mysql_databaseLine</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="3">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Password:</string>
                   </property>
                   <property name="buddy">
                    <cstring>mysql_passLine</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QSpinBox" name="mysql_portBox">
                   <property name="maximum">
                    <number>65535</number>
                   </property>
                   <property name="value">
                    <number>3306</number>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="mysql_databaseLine"/>
                 </item>
                 <item row="1" column="4">
                  <widget class="QLineEdit" name="mysql_passLine">
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="mysql_hostLine">
                   <property name="text">
                    <string>localhost</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Host:</string>
                   </property>
                   <property name="buddy">
                    <cstring>mysql_hostLine</cstring>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>databaseBox</tabstop>
  <tabstop>mysql_hostLine</tabstop>
  <tabstop>mysql_portBox</tabstop>
  <tabstop>mysql_userLine</tabstop>
  <tabstop>mysql_passLine</tabstop>
  <tabstop>mysql_databaseLine</tabstop>
  <tabstop>sqlite_fileLine</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
